workitem.2.go

```diff
@@ -96,7 +97,8 @@ func (c *Workitem2Controller) List(ctx *app.ListWorkitem2Context) error {
 
 	exp, err := query.Parse(ctx.Filter)
 	if err != nil {
-		return ctx.BadRequest(goa.ErrBadRequest(fmt.Sprintf("could not parse filter: %s", err.Error())))
+		jerrors, _ := jsonapi.ConvertErrorFromModelToJSONAPIErrors(goa.ErrBadRequest(fmt.Sprintf("could not parse filter: %s", err.Error())))
+		return ctx.BadRequest(jerrors)
```

Comment:
The controller has the last call on the overall HTTP error code and here it might indeed return a 500 whereas the internal JSONAPI error was something else. Well, theoretically. But it's fine to do it this way. I mean it's fine that the controller has the last call.